### PR TITLE
Update website for new committee

### PR DESCRIPTION
**Pull request by kumar-ish**
Changed the 'about' page to have all the new committee members. Also, changed 'challenging' to 'inclusive' as per the changes in the constitution :')

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
         <span class="role">Treasurer</span>
     </div>
     <div class="profile col s8 offset-s2 m6 l4">
-        <img class="profile-img" src="{{site.baseurl}}/img/exec/mgtlake.png">
-        <span class="name">Matthew Lake (<a href="https://uqcs.slack.com/messages/@mgtlake/" target="_blank">@mgtlake</a>)</span><br>
+        <a href="https://github.com/sannibunny">
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/sanni.jpg">
+        </a>
+        <span class="name">Sanni Bosamia (<a href="https://uqcs.slack.com/messages/@sanni/" target="_blank">@sanni</a>)</span><br>
         <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 2]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
     </div>
     <div class="profile col s8 offset-s2 m6 l4">
-        <img class="profile-img" src="{{site.baseurl}}/img/exec/que.png">
-        <span class="name">Ella de Lore (<a href="https://uqcs.slack.com/messages/@Ella/" target="_blank">@Ella</a>)</span><br>
+        <a href="https://github.com/jdcaperon">    
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/jack.jpg">
+        </a>
+        <span class="name">Jack Caperon (<a href="https://uqcs.slack.com/messages/@jam/" target="_blank">@jam</a>)</span><br>
         <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 3]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
     </div>
     <div class="profile col s8 offset-s2 m6 l4">
-        <a href="https://github.com/lighthou" target="_blank">
-          <img class="profile-img" src="{{site.baseurl}}/img/exec/ryan.jpg">
+        <a href="https://github.com/nicklambourne" target="_blank">
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/nick.png">
         </a>
-        <span class="name">Ryan Kurz (<a href="https://uqcs.slack.com/messages/@lighthou/" target="_blank">@lighthou</a>)</span><br>
+        <span class="name">Nicholas Lambourne (<a href="https://uqcs.slack.com/messages/@noodle/" target="_blank">@noodle</a>)</span><br>
         <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 4]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
     </div>
     <div class="profile col s8 offset-s2 m6 l4">
-        <img class="profile-img" src="{{site.baseurl}}/img/exec/ishraque.jpg">
-        <span class="name">Ishraque Zahin (<a href="https://uqcs.slack.com/messages/@Ishraque/" target="_blank">@Ishraque</a>)</span><br>
+        <a href="https://github.com/Moss1410" target="_blank">    
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/liv.png">
+        </a>
+        <span class="name">Olivia Mackenzie-Ross (<a href="https://uqcs.slack.com/messages/@Liv/" target="_blank">@Liv</a>)</span><br>
+        <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 5]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
+    </div>
+    <div class="profile col s8 offset-s2 m6 l4">
+        <a href="https://github.com/briwukaho/">
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/brian.png">
+        </a>
+        <span class="name">Brian Riwu Kaho (<a href="https://uqcs.slack.com/messages/@pipInstall/" target="_blank">@pipInstall</a>)</span><br>
+        <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 6]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
+    </div>
+    <div class="profile col s8 offset-s2 m6 l4">
+        <a href="https://github.com/coppers-j/">
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/james.jpg">
+        </a>
+        <span class="name">James Copperthwaite (<a href="https://uqcs.slack.com/messages/@jcopprthwaite/" target="_blank">@jcopprthwaite</a>)</span><br>
         <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

File: about.md [hunk 7]
```diff
@@ -26,53 +26,73 @@ We develop, hack and ultimately have fun.
...
     </div>
     <div class="profile col s8 offset-s2 m6 l4">
-        <a href="https://www.linkedin.com/in/joshua-sutton-753b10143/">
-          <img class="profile-img" src="{{site.baseurl}}/img/exec/jsutton.jpg">
+        <a href="https://github.com/KentonLam/">
+          <img class="profile-img" src="{{site.baseurl}}/img/exec/kenton.png">
         </a>
-        <span class="name">Joshua Sutton (<a href="https://uqcs.slack.com/messages/@jsutton/" target="_blank">@jsutton</a>)</span><br>
+        <span class="name">Kenton Lam (<a href="https://uqcs.slack.com/messages/@Kenton/" target="_blank">@Kenton</a>)</span><br>
         <span class="role">General Committee</span>
```
Comment (bradleysigma) [inline about.md]: Puisne Committee

**Comment by kumar-ish** [comment]
Not changing it to Puisne committee lol

**Pull request merged**